src/config/dynamic-theme-fixes.config

```diff
@@ -4481,6 +4481,15 @@ INVERT
 
 ================================
 
+celum.ssg-service.com
+
+CSS
+:hover {
```

Comment:
As I mentioned in comment this breaks hover color for hover over menu on the left 'cause selector here is too general.

Change it to `body:hover` and it will work like it should.